### PR TITLE
Small bugfixes, added config for environment.

### PR DESCRIPTION
**Pull request by elenakolevska**

### CODE REVIEW

File: client/modules/Slack/SlackBotSender.php [hunk 1]
```diff
@@ -41,8 +41,8 @@ class SlackBotSender
      */
     public function __construct($slackBotUrl, $auth = null)
     {
-        $this->slackBotUrl = $slackBotUrl;
-        $this->auth        = $auth;
+        self::$slackBotUrl = $slackBotUrl;
```
Comment (alebedev80) [inline client/modules/Slack/SlackBotSender.php:44]: @lv-dev so we can not have two instance of slackBotSender in process, i mean two client to two different slack bots.

Comment (elenakolevska) [reply client/modules/Slack/SlackBotSender.php:44]: Yes, I also thought of that, but I thought you guys consciously made that decision, cause these properties are defined as static in the class. And the compiler was throwing notices. @l80

File: client/index.php [hunk 2]
```diff
@@ -190,6 +189,13 @@
 try {
     $jiraWebhook->run();
 } catch (\Exception $e) {
+    // For convenience in local development show errors on screen directly
```
Comment (alebedev80) [inline client/index.php:192]: @lv-dev maybe better to use additional monolog handler in local env?
http://stackoverflow.com/questions/25787258/php-how-to-use-monolog-to-log-to-console-php-out

Comment (elenakolevska) [reply client/index.php:192]: Oh, I like this! thanks @alebedev80 . I'll check it out, see if we can log both in the log file and in stdout

File: client/modules/Slack/SlackBotSender.php (hunk 1 above)
Comment (alebedev80) [reply client/modules/Slack/SlackBotSender.php:44]: @lv-dev 
1. in the class was static properties - that is wrong
2. static properties should be deleted
3. so need to fix that code in another way, i think

now in the application can not be more than one client.

Comment (elenakolevska) [reply client/modules/Slack/SlackBotSender.php:44]: Ok, great @alebedev80 ! I'm changing it, even though we only have one client for now... Better do it now, while we're on it..

Comment (alebedev80) [reply client/modules/Slack/SlackBotSender.php:44]: @lv-dev i planned to do pull request of that client to SlackBot project

**Pull request merged**